cli/src/torii/routing.rs

```diff
@@ -57,7 +54,7 @@ impl VerifiedQueryRequest {
     pub fn validate(
         self,
         wsv: &WorldStateView,
-        query_validator: &IsQueryAllowedBoxed,
+        query_judge: &dyn Judge<Operation = QueryBox>,
```

Comment:
Just FYI, even though this is written as though it's a fat pointer to a `dyn` object, MIR is smart enough to devirtualise the call.

Reply:
Oh, that's cool